docs/src/components/ReanimatedTestimonials/ReanimatedTestimonialList/index.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState, useEffect } from 'react';
+import clsx from 'clsx';
+import styles from './styles.module.css';
+import ReanimatedTestimonialItem from '@site/src/components/ReanimatedTestimonials/ReanimatedTestimonialItem';
+
+const items = [
+  {
+    author: 'Marc Rousavy',
+    company: 'Margelo',
+    body: 'We’ve used Reanimated and Gesture Handler for a ton of apps already - it’s amazingly simple yet poweful! 😍',
+    link: 'https://twitter.com/mrousavy/status/1754909520571019756',
+    image: {
+      alt: 'marc rousavy',
+      src: 'img/testimonials/marc_rousavy.jpeg',
+    },
+  },
+  {
+    author: 'Andrew Lo',
+    company: 'Shopify',
+    body: 'They enable us app devs to make our users feel delight, have fun, and enjoy using our apps more.',
+    link: 'https://twitter.com/alotoronto/status/1754905332709576823',
+    image: {
+      alt: 'andrew lo',
+      src: 'img/testimonials/andrew_lo.jpeg',
+    },
+  },
+  {
+    author: 'Mo Gorhom',
+    body: `Reanimated and Gesture Handler are the reason why I shifted my focus from native (objc&java) development to React Native 🖤.`,
+    link: 'https://twitter.com/gorhom/status/1754974706782896465',
+    image: {
+      alt: 'mo gorhom',
+      src: 'img/testimonials/mo_gorhom.jpeg',
+    },
+  },
+  {
+    author: 'Brandon Austin',
+    body: `I’ve built dozens of apps, each and every one of them have used both Reanimated and Gesture Handler at different levels of complexity.`,
+    link: 'https://twitter.com/bran_aust/status/1754907731536863670',
+    image: {
+      alt: 'brandon austin',
+      src: 'img/testimonials/brandon_austin.jpeg',
+    },
+  },
+  {
+    author: 'Syed Noman',
+    body: `Reanimated is an amazing library!
+    It made my apps smooth and engaging, boosting performance and user happiness. Highly recommend for any developer wanting to upgrade  their app.`,
+    link: 'https://twitter.com/codewithnomi_/status/1754910812584436199',
+    image: {
+      alt: 'syed noman',
+      src: 'img/testimonials/syed_noman.jpeg',
+    },
+  },
+  {
+    author: 'Daniel Grychtol',
+    company: 'TheWidlarzGroup',
+    body: 'Simply - it’s pure joy to write animations with Reanimated now! Also, API that was introduced in V2 - imo game changer in terms of the learning curve (compared to API from V1) 💜.',
+    link: 'https://twitter.com/daniel_mark01/status/1754925618494337459',
+    image: {
+      alt: 'daniel grychtol',
+      src: 'img/testimonials/daniel_grychtol.jpeg',
+    },
+  },
+];
+
+const ReanimatedTestimonialList = () => {
```

Comment:
```suggestion
const TestimonialList = () => {
```
i don't think prefixing with `Reanimated` is necessary coz what else could it be 😄